templates/todo/projects/nodejs-mongo/.repo/terraform/infra/main.tf

```diff
@@ -1,7 +1,8 @@
 locals {
-  tags             = { azd-env-name : var.environment_name }
-  sha              = base64encode(sha256("${var.environment_name}${var.location}${data.azurerm_client_config.current.subscription_id}"))
-  resource_token   = substr(replace(lower(local.sha), "[^A-Za-z0-9_]", ""), 0, 13)
+  tags                         = { azd-env-name : var.environment_name }
+  sha                          = base64encode(sha256("${var.environment_name}${var.location}${data.azurerm_client_config.current.subscription_id}"))
+  resource_token               = substr(replace(lower(local.sha), "[^A-Za-z0-9_]", ""), 0, 13)
+  cosmos_connection_string_key = "AZURE-COSMOS-CONNECTION-STRING"

```

Comment:
We defined this in cosmos-account.bicep in bicep

Reply:
connection_key is not used in cosmos.tf as the key is pushed from the keyvault module and not the cosmos.tf